src/style.css

```diff
@@ -0,0 +1,108 @@
+/* #region html */
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-display: swap;
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-display: swap;
+  font-style: normal;
+  font-weight: 700;
+}
+
+* {
+  box-sizing: border-box;
+  padding: initial;
+
+  font-family: "Avenir", sans-serif;
+}
+/* #endregion */
+
+/* #region search-form */
+.search-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  position: relative;
+}
+
+.search-bar__content {
+  margin: 20px auto;
+}
+
+.input-search {
+  font-weight: 300;
+  line-height: 21.86px;
+  color: #3d4e61;
+  cursor: pointer;
+
+  background-color: #fff;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search-bar:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.input-search:focus {
+  font-weight: 700;
+  color: #3d4e61;
+
+  border-radius: 3px;
+  box-shadow: 0 4px 4px 0 #00000040;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  outline: none;
+}
+
+.search-bar__input {
+  font-size: 16px;
+
+  height: 70px;
+  padding-left: 62px;
+}
+
+.search-bar__input--small {
+  font-size: 14px;
+
+  height: 42px;
+  padding-left: 33px;
+}
+/* #endregion */
+
+/* #region icon */
+.search-bar::before {
```

Comment:
why did you set the background using before? 
why not directly to the input?

Reply:
good question....)

maybe practiced knowledge in using "before" and "after", BUT your tip significantly improved the code.
Thank you.